exasol/nb_connector/text_ai_extension_wrapper.py

```diff
@@ -0,0 +1,71 @@
+from typing import Optional
+
+from exasol.nb_connector.secret_store import Secrets
+from pathlib import Path
+
+LANGUAGE_ALIAS = "PYTHON3_TXAIE"
+
+LATEST_KNOWN_VERSION = "???"
+
+def deploy_licence(conf: Secrets,
+                   licence_file: Optional[Path] = None,
+                   licence_content: Optional[str] = None) -> None:
+    """
+    Deploys the given license and saves its identifier to the secret store. The licence can either be
+    defined by a path pointing to a licence file, or by the licence content given as a string.
+    Parameters:
+         conf:
+            The secret store.
+        licence_file:
+            Path of a licence file.
+        licence_content:
+            Content of a licence given as a string.
+
+    """
+    pass
+
+def initialize_text_ai_extension(conf: Secrets,
+                                 container_file: Path,
+                                 version: str = LATEST_KNOWN_VERSION,
+                                 language_alias: str = LANGUAGE_ALIAS,
+                                 run_deploy_container: bool = True,
+                                 run_deploy_scripts: bool = True,
+                                 run_upload_models: bool = True,
```

Comment:
We don't have scripts to deploy. Neither do we have any models to upload upfront.
But we probably need the BucketFS credentials and HF token, just like the TE.

Reply:
yes but we plan on having scripts and default models in the future, and want to prevent having to change the interface to often.

i am not sure about the HF token. that one is for private models, and i doubt we will have private models as default models for text ai, which are the only ones this installer will use. @tkilias what do you think?

Bucketfs credential you might be right. i thought those would be given through the secret store? is this not the case?

Reply:
User might use private models, so the token is needed

Reply:
Regarding bucketfs credentials that comes from the AI Lab config

Reply:
We will have scripts and default models. Scripts will be added in Q2 in form of span functions and default models, we actually will need already this quarter.

Reply:
having the HF token here still seems kinda iffy to me. this call does not need them. unless we allow users to set default model? which seem not default then.
otherwise if people want to install additional private models, should that not happen via a different call, which then can also be used to set the token?

Reply:
we can also remove the hf token, have no strong feelings about it